doc/vim-which-key.txt

```diff
@@ -292,7 +292,16 @@ If you wish to hide a mapping from the menu set its description to
 >
     nnoremap <leader>1 :1wincmd w<CR>
     let g:which_key_map.1 = 'which_key_ignore'
+>
+                                            *g:which_key_display_names*
+Type: |Dict|
+Default: `undefined`
```

Comment:
Default should be `{ ' ': 'SPC', '<C-H>': 'BS', '<C-I>': 'TAB', '<TAB>': 'TAB', }`.

Reply:
Right now it's not defined by default. The plugin checks if it exists and uses s:displaynames if it doesn't. Should I just make s:displaynames global, instead?

Reply:
Nope. It is enough to change the doc only.

```vim
Default: `{ ' ': 'SPC', '<C-H>': 'BS', '<C-I>': 'TAB', '<TAB>': 'TAB', }`
```

Reply:
Should be good now.